Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/webDriverSingleton/WebDriverSingleton.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.cdp.volha_ivanova.task1.webDriverSingleton;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebDriverSingleton {
+	private static WebDriver driver;
+
+	private WebDriverSingleton() {
+	}
+
+	public static WebDriver getWebDriverInstance() {
```

Comment:
getWebDriverInstance and initBrowser method should be combined. One point to get singleton object